lib/spack/spack/test/main.py

```diff
@@ -13,55 +12,42 @@
 import spack.util.git
 from spack.main import get_version, main
 
-pytestmark = pytest.mark.not_on_windows(
-    "Test functionality supported but tests are failing on Win"
-)
+git_exe_name = "git.bat" if sys.platform == "win32" else "git"
+script_head = "@echo off" if sys.platform == "win32" else "#!/bin/sh"
```

Comment:
Can we move this and the above into the method that uses them? I don't see them being used anywhere else.